config/desktop/bullseye/environments/mate/armbian/create_desktop_package.sh

```diff
@@ -5,19 +5,133 @@ cp -R "${SRC}"/packages/blobs/desktop/lightdm "${destination}"/etc/armbian
 mkdir -p "${destination}"/etc/skel
 cp -R "${SRC}"/packages/blobs/desktop/skel/. "${destination}"/etc/skel
 
-# install logo for login screen
-mkdir -p "${destination}"/usr/share/pixmaps/armbian
-cp "${SRC}"/packages/blobs/desktop/icons/armbian.png "${destination}"/usr/share/pixmaps/armbian
+#install cinnamon desktop bar icons
```

Comment:
"cinnamon" ?

Reply:
I cp that txt in while documenting
I will fix

Reply:
fixed refresh